lib/y2_logs_helper.pm

```diff
@@ -101,7 +101,7 @@ sub accept_license {
 }
 
 sub verify_license_translations {
-    return if (is_sle && get_var("BETA") || check_var('VIDEOMODE', 'text'));
+    return if (is_sle && get_var("BETA"));
```

Comment:
Note that SLE beta licenses are also translated meanwhile, so this could be changed as well.

Reply:
@Vogtinator change in what way?

Reply:
Also verify the translations for beta builds?

Reply:
no. translation is not verified in sle when is beta. this is the purpose of the condition

Reply:
it is broken and it is a known issue afaik

Reply:
It is not supposed to be broken, which is why this can and should be changed.

Reply:
The problem is that if translation is missing in beta, or is irrelevant, those bugs are rejected as it's too early to be reported. That's why this condition was there initially, so not much sense in testing functionality which is not expected to work, but might work.

Reply:
Beta licenses are translated now exactly because the switch to the final translated licenses was too late to find and fix bugs.